pubspec.lock

```diff
@@ -14,7 +14,7 @@ packages:
       name: async
```

Comment:
You will have to manually resolve the merge conflict here, and can't just use GitHub.

You will have to manually checkout this branch, do
git merge main
and resolve merge conflicts.

To resolve the conflicts for pubspeck.lock, just delete the file (.lock NOT .yaml) and rerun
flutter pub get
and it will re-generate the correct file.